src/sentry/remote_subscriptions/models.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+from enum import Enum
+from typing import ClassVar, Self
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import BaseManager, DefaultFieldsModel, region_silo_model
+
+
+@region_silo_model
+class RemoteSubscription(DefaultFieldsModel):
+    # TODO: This should be included in export/import, but right now it has no relation to
+    # any projects/orgs. Will fix this in a later pr
```

Comment:
How will the project/org relation work for this model?

Reply:
Via whichever service implements a more specific subscription. So for uptime, someone might make a subscription to monitor http://sentry.io. So they'll create a
`ProjectUptimeSubscription` - This ties an UptimeSubscription to a specific project
`UptimeSubscription` - this is generic, since we want to debounce monitors for the same url. So if multiple projects across orgs make the same monitor, they'll share this
`RemoteSubscription`

We'll need some kind of cleanup process that allows orphaned rows to be properly removed.